app/controllers/communities_controller.rb

```diff
@@ -96,10 +99,9 @@ def count_post_for_this_week
 
   def check_if_banned
     community = Community.find(params[:id])
-    puts current_account.id
     banned_user = BannedUser.find_by(account_id: current_account.id, community_id: community.id)
     unless banned_user.nil?
-      redirect_to '/404'
+      redirect_to '/302' and return
```

Comment:
shouldn't we show 404 page for banned user?

Reply:
banned Users must know they have been banned so it redirects to 403 ill change the redirection name and resolve it @iMagesh

Reply:
really? should we show people they are banned? check how it works in Reddit. Read their documentation and let me know.